juntagrico/admins/__init__.py

```diff
@@ -1,11 +1,22 @@
-from django.contrib import admin
+from juntagrico.admins.base_admin import BaseAdmin
 
-from juntagrico.util import addons
+from juntagrico.admins.member_admin import MemberAdmin, JuntagricoUserAdmin, MemberAdminWithShares
 
+from juntagrico.admins.share_admin import ShareAdmin
+
+from juntagrico.admins.subscription_admin import SubscriptionAdmin
+from juntagrico.admins.subscription_type_admin import SubscriptionTypeAdmin, SubscriptionSizeAdmin
+from juntagrico.admins.extra_subscription_admin import ExtraSubscriptionAdmin
+from juntagrico.admins.extra_subscription_type_admin import ExtraSubscriptionTypeAdmin
+
+from juntagrico.admins.job_admin import JobAdmin
+from juntagrico.admins.job_type_admin import JobTypeAdmin
+from juntagrico.admins.one_time_job_admin import OneTimeJobAdmin
+from juntagrico.admins.area_admin import AreaAdmin
+from juntagrico.admins.assignment_admin import AssignmentAdmin
+
+from juntagrico.admins.depot_admin import DepotAdmin
+from juntagrico.admins.list_message_admin import ListMessageAdmin
+from juntagrico.admins.delivery_admin import DeliveryAdmin
 
-class BaseAdmin(admin.ModelAdmin):
```

Comment:
wir sind hier mit python unterwegs und nicht in java, dh das baseadmin muss nicht in eine eigene datei das kann ganz ruhig hier bleiben. eigentlich sind auch die anderen admins nicht wirklich gut so wie sie sin sind. es würde eigentlich mehr sinn machen die ähnlich zu grupieren wie die entites, icht hatte das mal in eine anflug von javamässiger aufräum wut so gemacht bin aber nicht wirklich zufrieden damit